Live-Snap/Live-Snap/Utility/GenerateLiveWallpaperWithBarcode.swift

```diff
@@ -60,16 +60,10 @@ class GenerateLiveWallpaperWithBarcode {
     
     func interpolateFrames() -> [UIImage]? {
         
-        let frame1 = wallpaperImage
+        guard let frame2Background = wallpaperImage.darkenedAndBlurred(darkness: 0.06, blurRadius: 60) else { return nil }
+        guard let frame2 = drawSnapCodeOnImage(snapcode: barcodeImage, image: frame2Background) else { return nil }
         
-        guard let frame2Background = frame1.darkenedAndBlurred(darkness: 0.04, blurRadius: 16) else { return nil }
-        guard let blurredSnapcode = barcodeImage.blurred(blurRadius: 16) else { return nil }
-        guard let frame2 = drawSnapCodeOnImage(snapcode: blurredSnapcode, image: frame2Background) else { return nil }
-        
-        let frame3Background = frame1.darkenedAndBlurred(darkness: 0.06, blurRadius: 30)
-        guard let frame3 = drawSnapCodeOnImage(snapcode: barcodeImage, image: frame3Background!) else { return nil }
-        
-        return [frame1, frame2, frame3]
+        return [wallpaperImage, frame2]
     }
     
     func drawSnapCodeOnImage(snapcode: UIImage, image: UIImage) -> UIImage? {
```

Comment:
we may end up reusing this class in other projects, so you should rename anything in this file that uses "snap" or "snap code" references to "barcode"